### ISSUE TITLE
Saving task arguments would be very beneficial

### ISSUE THREAD

**Issue by VikasNeha** [boxine/django-huey-monitor#149]
In the task model, a lot of info is saved which is very useful. But in a production scenario, it will also be super helpful if the args passed to the task function are also stored. If a task fails, the error message is there but it becomes difficult to map it to the input. If you think this is useful, I would be willing to work on a PR for this.

**Comment by spapas** [comment]
I also agree with this, it is very useful and needs to be implemented! Right now, I get an exception like:

```
Unhandled exception in task 2e1f168e-3d51-44e6-8c5c-94faf83ea77c.

TypeError
object of type 'NoneType' has no len()
```

Now, althrough I can find the task with the id `2e1f168e-3d51-44e6-8c5c-94faf83ea77c` I can't really find its parameters to know how the task was called to be able to debug it.

So could we give some love to this PR?

cc @jedie

**Comment by jedie** [comment]
If we store the arguments, then a restart is also possible: https://github.com/boxine/django-huey-monitor/issues/34
